src/sentry/static/sentry/app/views/userFeedback/index.tsx

```diff
@@ -106,31 +110,62 @@ class OrganizationUserFeedback extends AsyncView<Props, State> {
   renderBody() {
     const {organization} = this.props;
     const {location} = this.props;
-    const {status} = getQuery(location.search);
+    const {pathname, search, query} = location;
+    const {status} = getQuery(search);
     const {reportListPageLinks} = this.state;
 
+    const unresolvedQuery = omit(query, 'status');
+    const allIssuesQuery = {...query, status: ''};
+
     return (
       <React.Fragment>
         <GlobalSelectionHeader organization={organization} />
         <PageContent>
           <LightWeightNoProjectMessage organization={organization}>
-            <UserFeedbackContainer
-              pageLinks={reportListPageLinks}
-              status={status}
-              location={location}
-            >
-              {this.renderStreamBody()}
-            </UserFeedbackContainer>
+            <div data-test-id="user-feedback">
+              <Header>
+                <PageHeading>{t('User Feedback')}</PageHeading>
```

Comment:
Saw some visual diffs from Percy because of some changed margins with the header, are we intentionally making the heading smaller?

Reply:
Yeah - Previously margin was on `PageHeading` causing the controls on the right to not be vertically aligned. The margin was also not using our grid units (`space()`), so this changes to use grid units.